src/components/Navbar/index.tsx

```diff
@@ -8,6 +8,59 @@ import { LinkInfo } from "./types"
 import WhatsNextBanner from "./WhatsNextBanner"
 import MobileDrawer from "./MobileNav/MobileDrawer"
 import DesktopNavLinks from "./DesktopNav/DesktopNavLinks"
+import { CardButton } from "../Card"
+import NavCTAButtons from "./NavCTAButtons"
+
+export const Navbar: FC = () => {
+  const {
+    isOpen: isDrawerOpen,
+    onOpen: onDrawOpen,
+    onClose: onDrawerClose,
+  } = useDisclosure()
+
+  const { isOpen: showBanner, onClose: closeBanner } = useDisclosure({
+    defaultIsOpen: true,
+  })
+  const data = useStaticQuery(query)
+  const socialLinks =
+    data.allMarkdownRemark.edges[0].node.frontmatter.social_links
+  const navLinks = data.allMarkdownRemark.edges[0].node.frontmatter
+    .nav_items as LinkInfo[]
+  const menuButtons = data.allMarkdownRemark.edges[0].node.frontmatter
+    .menu_buttons as CardButton[]
+
+  return (
+    <>
+      {showBanner && <WhatsNextBanner onClose={closeBanner} />}
+      <HStack
+        bg="gray.900"
+        h="90px"
+        borderBottom="1px solid"
+        borderColor="gray.700"
+        as="header"
+      >
+        <Container
+          maxW="1140px"
+          h="100%"
+          display="flex"
+          justifyContent="space-between"
+        >
```

Comment:
Non-blocking:

I think that the `NavBar` would require some refactoring/touch-ups, because in some resolutions it looks off. For example this is how it looks for `1173px` width:

<img width="461" alt="image" src="https://github.com/threshold-network/website/assets/40306834/4e52d2cd-d9fb-47f2-a34a-56680613b48a">

I think that setting the maxW to `1140px` with `justifyContent="space-between"` is a mistake. Content should probably just be justified to "center" and the width should be maxed out. But I'm not sure as of now.

I think this would require some more touch upd in the long run so I would say to address this issue in a separate PR. The bug is also happening on production (1126 px width):
<img width="271" alt="image" src="https://github.com/threshold-network/website/assets/40306834/38030e7e-78aa-4ea9-988e-5a5e47649661">

but adding the dashboard link in the navigation made it a little bit worse (see previous screen).

To not block this PR, I will mark it as non-blocking. Addressing that issue in this PR might extend the time to merge this one, as more code equals to more potential bugs that can be found 😅  So I would say we separate that into another PR. Let's just not forget about that issue..

You can test it out yourself and see it you can reproduce the issue.

Reply:
Good catch! I've opened an issue to keep track of that, will address it in a separate PR as advised. 
https://github.com/threshold-network/website/issues/107

Reply:
Great! I'll keep this conversation open just in case